modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/filename/PdsConsistentIdProcessor.java

```diff
@@ -122,22 +65,28 @@ public PdsConsistentIdProcessor(final GridKernalContext ctx) {
      * @param consistentId compatibility consistent ID
      * @return PDS folder settings compatible with previous versions.
      */
-    private PdsFolderSettings compatibleResolve(
+    private PdsFolderSettings<NodeFileLockHolder> compatibleResolve(
         @Nullable final File pstStoreBasePath,
         @NotNull final Serializable consistentId) {
 
         if (cfg.getConsistentId() != null) {
             // compatible mode from configuration is used fot this case, no locking, no consitent id change
-            return new PdsFolderSettings(pstStoreBasePath, cfg.getConsistentId());
+            return new PdsFolderSettings<>(pstStoreBasePath, cfg.getConsistentId());
         }
 
-        return new PdsFolderSettings(pstStoreBasePath, consistentId);
+        return new PdsFolderSettings<>(pstStoreBasePath, consistentId);
     }
 
     /** {@inheritDoc} */
-    @Override public PdsFolderSettings resolveFolders() throws IgniteCheckedException {
+    @Override public PdsFolderSettings<NodeFileLockHolder> resolveFolders() throws IgniteCheckedException {
         if (settings == null) {
-            settings = prepareNewSettings();
+            PdsFolderResolver<NodeFileLockHolder> resolver =
+                new PdsFolderResolver<>(cfg, log, ctx.discovery().consistentId(), this::tryLock);
```

Comment:
Should we use the `ClusterNode#consistentId` here and remove the deprecation? We already refactor this class.

Reply:
I think no.

According to javadoc comment I accidently removed it's used intentionally - 
```
//here deprecated method is used to get compatible version of consistentId
```

Reply:
I revert JavaDoc removal. Hope it becomes clear now, why we use deprecated method here.